tests/acceptance/features/bootstrap/NotificationContext.php

```diff
@@ -490,4 +500,63 @@ public function clearInbucketMessages():void {
 				$e->getMessage();
 		}
 	}
+
+	/**
+	 * @When admin creates a deprovisioning notification
+	 * @When user :user tries to create a deprovisioning notification
+	 *
+	 * @param string|null $user
+	 *
+	 * @return void
+	 */
+	public function userCreatesDeprovisioningNotification(?string $user = null):void {
+		$payload["type"] = "deprovision";
+		$payload["data"] = ["deprovision_date" => "2043-07-04T11:23:12Z"];
```

Comment:
can the date be dynamic? something like this:
```php
   $deprovisionDate = \date('Y-m-d\TH:i:s\Z', \strtotime("+30 days"));
```

Reply:
I'd keep it simple here. I tried do it dynamic but:
- I need check date in notification response. 
- for that I should set new function getDate() in substituteInLineCodes()
- need change getDate() if I choose different data format
- ...

Reply:
yeah, with dynamic date, checking it in response would be hard or need some store mechanism